Source/WebCore/dom/Document.cpp

```diff
@@ -10530,6 +10530,22 @@ void Document::setHasViewTransitionPseudoElementTree(bool value)
     m_hasViewTransitionPseudoElementTree = value;
 }
 
+bool Document::renderingIsSuppressedForViewTransition() const
+{
+    return m_renderingSuppressedForViewTransition;
+}
+
+void Document::setRenderingSuppressedForViewTransition(bool value)
+{
+    if (std::exchange(m_renderingSuppressedForViewTransition, value) && !value) {
+        if (CheckedPtr view = renderView())
+            view->compositor().setRenderingSuppressed(false);
```

Comment:
I don't like that the `setRenderingSuppressed(false)` is here, but the `setRenderingSuppressed(true)` is in a function on LocalFrameView. Please factor the code to put them next to each other.